cirq-core/cirq/transformers/dynamical_decoupling.py

```diff
@@ -177,7 +177,17 @@ def _calc_pulled_through(
     clifford_ops_in_moment: list[ops.Operation] = [
         op for op in moment.operations if _is_clifford_op(op)
     ]
-    return input_pauli_ops.after(clifford_ops_in_moment)
+    ps_after: ops.PauliString = ops.PauliString()
+    try:
+        ps_after = input_pauli_ops.after(clifford_ops_in_moment)
+    except TypeError:
+        # PauliString.after might fail due to a known decomposition issue.
+        # TODO(#): remove the except handling after # is dixed.
```

Comment:
@NoureldinYosri  wonder if could you help point to the related issue id? I didn't find the matching issue from the issue list.

Reply:
there is not an issue atm. can you create one?

Reply:
done filing #6946